js/src/hooks/useAdsCampaigns.js

```diff
@@ -24,34 +25,45 @@ const selectorName = 'getAdsCampaigns';
  * A hook that calls `getAdsCampaigns` selector to load current campaigns
  * from merchant's Google Ads account if connected.
  *
+ * @param {...*} args Arguments to be forwarded to the selector. The only allowed argument is {exclude_removed: true|false}
```

Comment:
💅  This could be clearer using the explicit available params as well as changing from 'args' to 'query' since `query` it's the name of the param that `getAdsCampaigns` will receive as an argument. In case the available query params grows in the future we could consider also creating a typeDef kinda `getAdsCampaignsQuery` with the available params.

```
@param {{exclude_removed: true|false}} query Query to be forwarded to the selector.
const useAdsCampaigns = ( ...query ) => {
```

Reply:
Thanks for the suggestion, I added here: [f7620a9](https://github.com/woocommerce/google-listings-and-ads/pull/1505/commits/f7620a97f8216ce2a0db67dd972bbe75a3a0f79a)